templates/playground/src/skills/search.ts

```
@@ -79,34 +84,35 @@ async function searchDuckDuckGo(query: string): Promise<string[]> {
-    
+
     const html = await response.text();
-    
+
     // Extract URLs from the results
     const urlRegex = /<a class="result-link" href="([^"]+)"/g;
     const urls: string[] = [];
     let match;
-    
+
     while ((match = urlRegex.exec(html)) !== null) {
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)